cmd/neofs-node/config.go

```diff
@@ -155,10 +156,11 @@ func (c *shardCfg) id() string {
 
 type subStorageCfg struct {
 	// common for all storages
-	typ   string
-	path  string
-	perm  fs.FileMode
-	depth uint64
+	typ    string
+	path   string
+	perm   fs.FileMode
+	depth  uint64
+	noSync bool
```

Comment:
Is it really common of FSTree-specific?

Reply:
+1 for adding a new block with a comment for FSTree-specific, it just did not exist when i wrote that part

Reply:
Currently is _can_ be used for both fstree and blobovnicza. I think it is common -- every storage type can provide different guarantees.